File: plugins/devkit/devkit-java-tests/testData/inspections/useGrayFix/UseGrayConstantFixInJBColorParam.java
import com.intellij.ui.JBColor;
import com.intellij.ui.Gray;

import java.awt.Color;

class UseGrayConstantFixInJBColorParam {

  void any() {
    Color myGray = new JBColor(<warning descr="'java.awt.Color' used for gray">new Co<caret>lor(25, 25, 25)</warning>, Gray._125);
  }

}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/useGrayFix/UseGrayConstantFixInJBColorParam.kt
import com.intellij.ui.Gray
import com.intellij.ui.JBColor
import java.awt.Color

class UseGrayConstantFixInJBColorParam {
  fun any() {
    @Suppress("UNUSED_VARIABLE")
    val myGray: Color = JBColor(<warning descr="'java.awt.Color' used for gray">Col<caret>or(25, 25, 25)</warning>, Gray._125)
  }
}
